Firestore/core/test/firebase/firestore/remote/serializer_test.cc

```diff
@@ -14,24 +14,15 @@
  * limitations under the License.
  */
 
-/* NB: proto bytes were created via:
-     echo 'TEXT_FORMAT_PROTO' \
-       | ./build/external/protobuf/src/protobuf-build/src/protoc \
-           -I./Firestore/Protos/protos \
-           -I./build/external/protobuf/src/protobuf/src \
-           --encode=google.firestore.v1beta1.Value \
-           google/firestore/v1beta1/document.proto \
-       | hexdump -C
- * where TEXT_FORMAT_PROTO is the text format of the protobuf. (go/textformat).
+/* NB: Most tests use libprotobuf to create the bytes used for testing the
```

Comment:
style nit: I think you can drop the "NB" here and possibly elsewhere in this PR. When nearly every comment has this marking this no longer has the intended effect (https://www.youtube.com/watch?v=A8I9pYCl9AQ).

If there is one part of this that's worth highlighting, it's the line about how libprotobuf is only used in the test suite. The rest of the comment is just a comment.